### PR TITLE
docs: added documentation folder for extensions

### PR DESCRIPTION
**Pull request by dackers86**
Storage/implementation of official extensions documentation

fixes: https://github.com/firebase/extensions/issues/949
fixes: https://github.com/firebase/extensions/issues/963

### CODE REVIEW

File: .docs/firestore-send-email/ManageDeliveryStatus.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Manage delivery status
+
+Created: April 27, 2022 4:30 PM
+Last Edited Time: April 27, 2022 4:37 PM
+Status: Approved
```
Comment (Salakar) [inline .docs/firestore-send-email/ManageDeliveryStatus.md:5]: Remove

File: .docs/firestore-send-email/SMTPConnectionURL.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# SMTP connection URL
+
+Created: April 27, 2022 4:38 AM
+Last Edited Time: April 29, 2022 12:28 PM
+Status: In Progress
```
Comment (Salakar) [inline .docs/firestore-send-email/SMTPConnectionURL.md:5]: Remove

File: .docs/firestore-send-email/UseHandlebarsTemplates.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
+# Use Handlebars templates
+
+Created: April 27, 2022 4:28 PM
+Last Edited Time: April 27, 2022 4:36 PM
+Status: Approved
```
Comment (Salakar) [inline .docs/firestore-send-email/UseHandlebarsTemplates.md:5]: Remove

**Comment by Salakar** [comment]
Lets rename `.docs` to `docs` also

File: docs/firestore-send-email/GetStarted.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
+# Get started
+
+## Using the Trigger Email extension
+
+The Trigger Email extension (`firestore-send-email`) lets you automatically send emails based on documents in a Cloud Firestore collection. Adding a document to the collection triggers this extension to send an email built from the document's fields. The document's top-level fields specify the email sender and recipients, including `to`, `cc`, and `bcc` options (each supporting UIDs). The document's `message` field specifies the other email elements, like the subject line and email body (either plaintext or HTML).
+
+Here's a basic example document write that would trigger this extension:
+
+```jsx
```
Comment (Ehesp) [inline docs/firestore-send-email/GetStarted.md:9]: `js` vs `jsx`

**Comment by i14h** [comment]
@kevinthecheung can you please take a look at these docs? we intend to merge them with firebase documentation as well

**Review by Ehesp — CHANGES_REQUESTED** [review]

**Review by kevinthecheung — CHANGES_REQUESTED** [review]

**Comment by dackers86** [comment]
Thanks @kevinthecheung. Suggestions committed.

**Review by kevinthecheung — APPROVED** [review]

File: docs/firestore-bigquery-export/CrossProjectSupport .md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+# Cross-Project Support
```
[the remark this answers is not in the document]
Comment (dackers86) [reply docs/firestore-bigquery-export/CrossProjectSupport .md]: Thanks!

**Review by Ehesp — APPROVED** [review]

**Pull request merged**